src/flick/lst/models.py

```diff
@@ -3,16 +3,18 @@
 from django.db import models
 
 from show.models import Show
+from user.models import Profile
 
 
 class Lst(models.Model):
     lst_name = models.CharField(max_length=100)
-    lst_pic = models.ForeignKey(AssetBundle, on_delete=models.CASCADE, blank=True, null=True)
-    is_favorite = models.BooleanField(default=False)
-    is_private = models.BooleanField(default=False)
-    is_watched = models.BooleanField(default=False)
-    collaborators = models.ManyToManyField(User, related_name="collaborators", blank=True)
-    owner = models.ForeignKey(User, related_name="owner", on_delete=models.CASCADE)
+    lst_pic = models.TextField(blank=True, null=True)
+    # lst_asset_bundle = models.ForeignKey(AssetBundle, on_delete=models.CASCADE, blank=True, null=True)
+    is_favorite = models.BooleanField(default=False, null=True)
```

Comment:
Just wondering is is_favorite per list or per user per lst

Reply:
ah i guess we need to make that more clear. for now we can leave this here but we'll have to discuss 1) how to generate the default favorite list 2) put in extra measures to make sure you can't make more favorite lists